### PR TITLE
NewWithBracesFixer - End tags 

### PR DESCRIPTION
**Pull request by SpacePossum**
Little bug fix for the `NewWithBracesFixer`;
Input
```php
<?php $a = new \Exception?> <?php echo 1;
```
gets fixed to;
```php
<?php $a = new \Exception?> <?php echo 1();
```
which breaks valid code.

Additional fix in this PR is for;
input:
```php
<?php $x = new Z /**/;//
```

get fixed to:
```php
<?php $x = new Z() /**/;//
```

### CODE REVIEW

File: Symfony/CS/Fixer/Symfony/NewWithBracesFixer.php [hunk 1]
```diff
@@ -26,15 +26,15 @@ class NewWithBracesFixer extends AbstractFixer
     public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
-
-        for ($index = $tokens->count() - 1; $index >= 0; --$index) {
+        $count = $tokens->count();
```
Comment (SpacePossum) [inline Symfony/CS/Fixer/Symfony/NewWithBracesFixer.php]: I'll revert this little debug session leftover

**Comment by keradus** [comment]
:+1:, but 1.11.x

File: Symfony/CS/Fixer/Symfony/NewWithBracesFixer.php [hunk 2]
```diff
@@ -26,15 +26,14 @@ class NewWithBracesFixer extends AbstractFixer
     public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
-
```
Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Symfony/NewWithBracesFixer.php]: ?

Comment (SpacePossum) [inline Symfony/CS/Fixer/Symfony/NewWithBracesFixer.php]: thanks, reverted :+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**